tests/trainer/test_trainer.py

```diff
@@ -1504,6 +1504,29 @@ def test_auto_batch_size_finder(self):
         with patch.object(sys, "argv", testargs):
             run_glue.main()
 
+    def test_auto_batch_size_with_resume_from_checkpoint(self):
+        train_dataset = RegressionDataset(length=128)
+
+        config = RegressionModelConfig(a=0, b=2)
+        model = RegressionRandomPreTrainedModel(config)
+
+        tmp_dir = self.get_auto_remove_tmp_dir()
+        args = RegressionTrainingArguments(
+            tmp_dir,
+            do_train=True,
+            max_steps=2,
+            save_steps=1,
+            per_device_train_batch_size=8,
+            auto_find_batch_size=True,
```

Comment:
I don't think I know enough about `auto_find_batch_size` to understand the implication of this test. If `auto_find_batch_size` is `True` and `per_device_train_batch_size` is set - which one takes precedence?